docs/source/guides/develop-iroha-module.md

```diff
@@ -40,7 +40,7 @@ All public entites of your module should be documented as well. But first, let's
 
 ### 3. Write your logic
```

Comment:
This guide should be revised. Preferably once we have decided the shape in which modules will be implemented.

Reply:
sure

doc issue to track this: https://github.com/hyperledger/iroha-2-docs/issues/203